WebApp/autoreduce_webapp/selenium_tests/tests/base_tests.py

```diff
@@ -44,9 +45,10 @@ def tearDown(self) -> None:
 
     def _screenshot_driver(self):
         now = datetime.datetime.now()
-        screenshot_name = f"{self._testMethodName}-{now.strftime('%Y-%m-%d_%H-%M-%S')}.png"
-        path = str(Path(PROJECT_ROOT, "WebApp", "autoreduce_webapp", "selenium_tests", "screenshots", screenshot_name))
-        self.driver.save_screenshot(path)
+        screenshot_name = f"{type(self).__name__}-{self._testMethodName}-{now.strftime('%Y-%m-%d_%H-%M-%S')}.png"
+        report_path = str(
```

Comment:
why report path?

Reply:
Didn't see that. I'll change it to `screenshot_path`